mkat_fpga_tests/test_cbf.py

```diff
@@ -604,44 +604,32 @@ def get_actual_phases():
 
         actual_phases = get_actual_phases()
         expected_phases = get_expected_phases()
+        file_name = 'Delay_Phases_Response.svg'
+        units = 'secs'
+        expected_phases = [phase for rads, phase in get_expected_phases()]
+        aqf_plot_phase_results(no_chans, actual_phases, expected_phases,
+                                units, file_name, title)
+
+        actual = [phases for delays, phases in actual_phases]
+
         for i, delay in enumerate(test_delays):
-            delta_actual = round(np.max(actual_phases[i][1]) - np.min(
-                actual_phases[i][1]),2)
+            delta_actual = round(np.max(actual[i]) - np.min(actual[i]),2)
```

Comment:
Why are you round()-ing the values? You must compute them exactly and then use an is-close test. Don't round the data before comparing.